src/utils/dateTime.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Let's rename this file to `utils.ts` to make it more generic, for now I don't assume more date time related util methods